src/styles/modules/Title.scss

```diff
@@ -0,0 +1,5 @@
+.app__title {
+    font-size: 40px;
+    font-weight: 700;
+    text-shadow: $text-shadow;
+}
```

Comment:
empty line on unix systems at the end of the file is required. As you are using vs code, please install this package:
jdforsythe.add-new-line-to-files.
It add empty line to the file, on save, when there is no such a line. 
Please, improve all the files with that.